src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -107,15 +113,20 @@
     /** The fold by which the baited region has been amplified above genomic background. */
     public double FOLD_ENRICHMENT;
 
-    /** The number of targets that did not reach coverage=2 over any base. */
+    /** The number of targets that did not reach coverage1 over any base. */
     public double ZERO_CVG_TARGETS_PCT;
 
+    /** The fraction of aligned bases that were filtered out because they were in reads with low mapping quality (default is < 0). */
+    public double PCT_EXC_MAPQ;
```

Comment:
Why include a new metric for this, but not for the pct excluded for base q or overlapping reads?